meta-intel-edison-distro/recipes-core/images/edison-image-minimal.bb

```diff
@@ -89,3 +89,12 @@ IMAGE_INSTALL_append = " libgpiod"
 IMAGE_INSTALL_append = " htop"
 
 DEPENDS += "u-boot"
+
+ROOTFS_POSTPROCESS_COMMAND += "install_initrd; "
+
+install_initrd() {
+    bbnote "Adding initrd to image ${IMAGE_ROOTFS}"
+    install -d {IMAGE_ROOTFS}/boot
+    bbnote "from ${DEPLOY_DIR_IMAGE}/core-image-minimal-initramfs-edison.cpio.gz"
+    install -m 0755 ${DEPLOY_DIR_IMAGE}/core-image-minimal-initramfs-edison.cpio.gz ${IMAGE_ROOTFS}/boot/initrd
```

Comment:
Why `0755`?

Reply:
Really don't know. I copied from an example. But eventually it ends up on the image `edison-image-edison.hddimg` which installs into the boot partition. And that contains:
```
-rwxr-xr-x 1 root root 8536576 mei 30 18:41 bzImage
-rwxr-xr-x 1 root root 9553849 mei 30 18:41 initrd
...
```
So at least it's consistent with the kernel permissions :-) What would you prefer?